gossip/src/crds.rs

```diff
@@ -242,6 +242,11 @@ impl Crds {
                     CrdsData::DuplicateShred(_, _) => {
                         self.duplicate_shreds.insert(value.ordinal, entry_index);
                     }
+                    CrdsData::ContactInfo(node) => {
+                        // Can't insert into self.nodes since not all validators support ContactInfo yet
+                        // Querying self.nodes (get_nodes_contact_infos()) assumes we're pulling LegactContactInfo
+                        self.shred_versions.insert(pubkey, node.shred_version());
```

Comment:
hmmm, wondering if we should hold off updating `self.shred_versions` for now until we have switched to use the new contact-info.
e.g. you may get legacy and new contact-infos from the same node with different shred-versions.